[Detection Engine][Exceptions] - Update docs for single and shared exceptions (#4021)
elastic/security-docs: docs/detections/api/rules/rules-api-overview.asciidoc

returns alerts, and updates their statuses
* `<kibana host>:<port>/api/detection_engine/rules/prepackaged` - Loads and retrieves
the status of Elastic <<prebuilt-rules, prebuilt rules>>

> * <kibana host>:<port>/api/detection_engine/<rule_id>/exceptions` -  Creates a default exception list for the rule you specify.

Author: ```suggestion
* `<kibana host>:<port>/api/detection_engine/<rule_id>/exceptions` -  Creates a default exception list for the rule you specify.
```

returns alerts, and updates their statuses
* `<kibana host>:<port>/api/detection_engine/rules/prepackaged` - Loads and retrieves
the status of Elastic <<prebuilt-rules, prebuilt rules>>
* `<kibana host>:<port>/api/detection_engine/<rule_id>/exceptions` - Creates a default exception list for the rule you specify

> * `<kibana host>:<port>/api/detection_engine/rules/<id>/exceptions` - Creates exception items for the rule you specify 

Reviewer: Just checking — does it create exception items for the specified rule, or would it be more accurate to say for the rule's default exception list?

Author: In this case, the latter would be more accurate. I'll fix it - thanks!
```suggestion
* `<kibana host>:<port>/api/detection_engine/rules/<id>/exceptions` - Creates exception items for a rule's default exception list. 
```
